src/components/Order/Order.tsx

```diff
@@ -307,6 +318,8 @@ class Order extends React.Component<OrderProps, OrderState> {
       ? baseAssetAccuracy
       : quoteAssetAccuracy;
 
+    const enoughLiquidity = marketTotalPrice !== undefined;
```

Comment:
const enoughLiquidity = !!marketTotalPrice;

Reply:
We have to leave this check because marketTotalPrice can also be 0